examples/angular2/index.html

```diff
@@ -5,10 +5,18 @@
 		<title>Angular2 • TodoMVC</title>
 		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
 		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
-		<script src="node_modules/angular2/bundles/angular2-polyfills.js"></script>
-		<script src="node_modules/systemjs/dist/system.src.js"></script>
-		<script src="node_modules/rxjs/bundles/Rx.js"></script>
-		<script src="node_modules/angular2/bundles/angular2.dev.js"></script>
+		
+		<!-- Polyfill(s) for older browsers -->
```

Comment:
Why is the indentation for this comment inconsistent with the next line where you ref the shims?